src/QueryType/Update/Query/Query.php

```diff
@@ -44,14 +45,19 @@ class Query extends BaseQuery
     const COMMAND_COMMIT = 'commit';
 
     /**
-     * Update command rollback.
+     * Update command optimize.
      */
-    const COMMAND_ROLLBACK = 'rollback';
+    const COMMAND_OPTIMIZE = 'optimize';
```

Comment:
why did you delete 'rollback'?

Reply:
I see, you changed the order.

Reply:
What would be the best order? There is no single correct 'natural flow' for these commands. Should I change them to alphabetical order? The order used in `RequestBuilder`'s `switch/case`?

Reply:
I went with alphabetical. That's more or less consistent with other `const`s in the codebase.